<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VegeDialog</class>
 <widget class="QDialog" name="VegeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>261</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>新鲜的蔬菜！！！</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>在前面的选项框中打钩就行了</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>400</y>
     <width>721</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="refusePushButton">
      <property name="text">
       <string>清空</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="returnPushButton">
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="payPushButton">
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QSpinBox" name="qincaiSpinBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
   <property name="suffix">
    <string>/斤</string>
   </property>
   <property name="singleStep">
    <number>5</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="qingcaiSpinBox">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>220</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
   <property name="suffix">
    <string>/斤</string>
   </property>
   <property name="singleStep">
    <number>5</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="carrotSpinBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
   <property name="suffix">
    <string>/斤</string>
   </property>
   <property name="singleStep">
    <number>5</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="tomatoSpinBox">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>280</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
   <property name="suffix">
    <string>/斤</string>
   </property>
   <property name="singleStep">
    <number>5</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="baicaicarrotCheckBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>白菜</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="kuguaCheckBox">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>350</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>苦瓜</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="kuguaSpinBox">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>350</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
   <property name="suffix">
    <string>/斤</string>
   </property>
   <property name="singleStep">
    <number>5</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="baicaiSpinBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>280</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
   <property name="suffix">
    <string>/斤</string>
   </property>
   <property name="singleStep">
    <number>5</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>361</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="picture.qrc">:/new/prefix0/resource/cai.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="qincaiCheckBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>芹菜</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="qingcaiCheckBox">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>220</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>青菜</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="carrotCheckBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>胡萝卜</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="tomatoCheckBox">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>280</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>西红柿</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="picture.qrc"/>
 </resources>
 <connections/>
</ui>
